ui/tests/helpers/clients.js

```diff
@@ -58,7 +58,7 @@ export function generateConfigResponse(overrides = {}) {
     request_id: 'some-config-id',
     data: {
       default_report_months: 12,
-      enabled: 'default-enable',
```

Comment:
FYI, when ENT vault is started this value is `default-enable` from the API until it's changed, and likewise for OSS and `default-disable`. We should make sure we don't lose support for those values (unless the API removed them and I wasn't aware)

Reply:
Oh! I has looked and saw it was `enable` but I believe that was when I had turned it off and back on....I'll change back!